server/src/entities/album-user.entity.ts

```diff
@@ -0,0 +1,31 @@
+import { AlbumEntity } from 'src/entities/album.entity';
+import { UserEntity } from 'src/entities/user.entity';
+import { Column, Entity, Index, JoinColumn, ManyToOne, PrimaryColumn } from 'typeorm';
+
+export enum AlbumUserRole {
+  Editor = 'editor',
+  Viewer = 'viewer',
```

Comment:
Enums in the server are all caps right now, even though I think I might prefer to move to this style. Can you make them all caps to be consistent though?

Reply:
Actually I don't care if you change this or not. I'm going to start using this format myself I think.